agent/agent-tooling/src/main/java/io/opentelemetry/sdk/metrics/internal/view/UserAgents.java

```diff
@@ -3,13 +3,13 @@
 
 package io.opentelemetry.sdk.metrics.internal.view;
 
-import io.opentelemetry.api.common.Attributes;
-import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import com.microsoft.applicationinsights.agent.internal.init.AiContextKeys;
+import io.opentelemetry.context.Context;
 
 final class UserAgents {
 
-  static boolean isBot(Attributes attributes) {
-    String userAgent = attributes.get(SemanticAttributes.HTTP_USER_AGENT);
+  static boolean isBot(Context context) {
+    String userAgent = context.get(AiContextKeys.USER_AGENT);
```

Comment:
same here, I thought the only change needed would be `HTTP_USER_AGENT` -> `USER_AGENT_ORIGINAL`